src/pages/Dashboard.vue

```diff
@@ -161,12 +239,38 @@
                 <p class="h-4 w-1/2 place-self-center bg-yellow-900 rounded-md mt-1"></p>
               </div>
               <div v-else>
+                <!-- value -->
                 <p :class="cardMetricValueClass">
                   {{ numQuestionsAnswered }}
                 </p>
-                <p :class="cardMetricTitleClass">
-                  {{ $t("dashboard.summary.num_questions_answered") }}
-                </p>
+                <div :class="cardMetricTitleClass" class="flex justify-center">
+                  <div
+                    class="flex flex-col md:flex-row space-y-2 md:space-y-0 md:space-x-2 items-center px-1"
+                  >
+                    <!-- title -->
+                    <p>
```

Comment:
```suggestion
                    <p class="leading-4">
```
Without this the metric title is not centered with respect to the  `i` icon